python/gui/auto_generated/symbology/qgssvgselectorwidget.sip.in

```diff
@@ -138,6 +138,43 @@ Defines if the group box to fill parameters is visible
 Returns if the group box to fill parameters is visible
 
 .. versionadded:: 3.18
+%End
+
+    void setBrowserVisible( bool visible );
+%Docstring
+Defines if the SVG browser should be visible
+
+.. versionadded:: 3.20
+%End
+
+    bool browserVisible() const;
+%Docstring
+Returns if the SVG browser should be visible
+
+.. versionadded:: 3.20
+%End
+
+    void setAllowAnyImage( bool allowAnyImage );
+%Docstring
+Defines if the widget should allow any image
```

Comment:
This isn't very explanatory - what is "any image"? Does that include non-svg formats?